scripts/overview.py

```diff
@@ -40,10 +43,21 @@ def get_summary(
             np.sum(petab_problem.parameter_df[petab.ESTIMATE]),
         'events':
             len(petab_problem.sbml_model.getListOfEvents()),
+        'preequilibration':
+            'No' if petab.PREEQUILIBRATION_CONDITION_ID not in
+            petab_problem.measurement_df.columns or
+            pd.isnull(petab_problem.measurement_df[
+                          petab.PREEQUILIBRATION_CONDITION_ID]).all()
+            else 'Yes',
+        'postequilibration':
+            'Yes' if np.isinf(petab_problem.measurement_df[petab.TIME]).any()
+            else 'No',
```

Comment:
Is the `TIME` column always of a suitable type? i.e. if the column type is "object" instead of "float", then maybe `inf` is stored as type `str` and `np.isinf` doesn't identify `'inf'` as equivalent to `np.inf`. The `libpetab-python` library currently doesn't appear to cast `'inf'` to float in the `TIME` column of a measurements table in the `get_measurement_df` method at least [1]. This would need to be fixed before this result can be trusted.

https://github.com/PEtab-dev/libpetab-python/blob/main/petab/measurements.py#L28-L50